eng/Directory.Build.Common.targets

```diff
@@ -49,7 +49,7 @@
   <PropertyGroup>
     <OriginalReadmeMdPath>$([MSBuild]::NormalizeDirectory($(MSBuildProjectDirectory)/../))README.md</OriginalReadmeMdPath>
     <ProcessedReadmeMdPath>$(IntermediateOutputPath)README.md</ProcessedReadmeMdPath>
-    <EnableNuGetReadmeMd Condition="'$(HasReleaseVersion)' == 'false' and Exists('$(OriginalReadmeMdPath)')">true</EnableNuGetReadmeMd>
+    <EnableNuGetReadmeMd Condition="'$(IsDesignTimeBuild)' == 'false' and Exists('$(OriginalReadmeMdPath)')">true</EnableNuGetReadmeMd>
```

Comment:
Shouldn't we keep both conditions?

Reply:
We had the `HasReleaseVersion` to only include README in previews, so removing it accomplishes having readme in GA packages as well.